README.md

```diff
@@ -40,6 +40,16 @@ Aragonette is built to work seamlessly with Aragon OSx primitives, such as `IPro
 
 Got ideas on how to make Aragonette even better? We're all ears! Whether it's a bug fix, a new feature, or a plugin that could benefit everyone, we welcome your contributions. Check out our [contributing guidelines](CONTRIBUTING.md) for more information on how to get involved.
 
+### You can configure your repository to pull changes from this repository with:
+```
+git remote add upstream git@github.com:aragon/aragonette.git
+git remote set-url --push upstream DISABLE
+
+# Then you can do
+git pull
+git push upstream <your branch>
```

Comment:
Wouldn't this command be unsuccessful since the second one disables the pushes to the upstream (main repo)? Considering the documentation is for pulling changes from upstream, I'm not sure we need to include this. Alternatively, if we want, we can have another subsection explaining how to open a PR on the main repo from a fork. Thoughts?

Reply:
You're correct. Initially, I thought it would only disable non-explicit pushes, but that's not the case. I'll remove it.